File: documentation/user/en/query/ordering/examples/natural/attribute-natural-multiple.cs
EvitaResponse<SealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	OrderBy(
        		AttributeNatural("ean", Asc),
        		AttributeNatural("catalogNumber", Desc)
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code", "ean", "catalogNumber")
        		)
        	)
        )
	)
);

File: documentation/user/en/query/ordering/examples/comparable/attribute-natural-multiple.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				orderBy(
					attributeNatural("ean", ASC),
					attributeNatural("catalogNumber", DESC)
				),
				require(
					entityFetch(
						attributeContent("code", "ean", "catalogNumber")
					)
				)
			)
		);
	}
);
